test/verify/check-system-realms

```diff
@@ -672,7 +672,7 @@ ipa-advise enable-admins-sudo | sh -ex
         m.execute(f'''! selinuxenabled || semanage port -m -t websm_port_t -p tcp 443
                       mkdir -p /run/systemd/system/cockpit.socket.d/
                       printf "[Socket]\nListenStream=\nListenStream=443" > /run/systemd/system/cockpit.socket.d/listen.conf
-                      sed -i '/\[WebService/ aOrigins = http://{m.web_address}:{m.web_port}' /etc/cockpit/cockpit.conf''')
+                      sed -i '/[WebService/ aOrigins = http://{m.web_address}:{m.web_port}' /etc/cockpit/cockpit.conf''')
```

Comment:
Hmm, isn't the intention here to pass `\[` to sed? (so that it matches a literal `[` instead of starting a character class.)

Reply:
Yes, `f'''\['''` produces `\[`, and to silence the warning without changing the result, this needs to be `f'''\\['''`